kahuna/public/stylesheets/main.css

```diff
@@ -306,32 +306,70 @@ textarea.ng-invalid {
 /* ==========================================================================
    Top bar
    ========================================================================== */
+@media screen and (max-width: 700px) {
+    .top-bar-item.top-bar-item--surplus {
+        display: none;
+    }
+}
+   
+@media screen and (max-width: 1100px) {
+    .top-bar-item--label-text {
+        display: none;
+    }
+    .top-bar-item .archiver--label-text {
+        display: none;
+    }
```

Comment:
should we combine this rule with the above one? Is `archiver--label-text` used? I can't seem to find it in any of the templates.